website/content/en/docs/advanced-topics/multi-arch.md

```diff
@@ -36,23 +41,177 @@ docker buildx build --push --platform linux/amd64,linux/arm64,linux/ppc64le,linu
+**Caveat**: the Dockerfile generated by the SDK for the operator explicitly references GOARCH=amd64 for go build. This can be amended to GOARCH=$TARGETARCH. Docker will automatically set the environment variable to the value specified by --platform. With buildah --build-arg will need to be used for the purpose.
 
-**Caveats**: When mirroring registries for disconnected installations (environments without internet connection) all the images referenced by a manifest list need to be copied, including images for architectures that may not be used in the environment.
+**Caveat**: When mirroring registries for disconnected installations (environments without internet connection) all the images referenced by a manifest list need to be copied, including images for architectures that may not be used in the environment.
 
-#### Operator Lifecycle Manager
+### Operator Lifecycle Manager
 
 For operators distributed through the [Operator Lifecycle Manager (OLM)][olm]:
 
 * [Bundle images][bundle] are not architecture-specific. They contain only plaintext kubernetes manifests and operator metadata.
 * All image references in the ClusterServiceVersion should be manifest lists containing the pointers to the image manifests for the supported architectures.
 * Labels for OS and architectures can be set in the CSV. Please refer to the [Operator Lifecycle Management Documentation][olm_multiarch] for details.
 
-[manifest_list]: https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list
-[image_index]: https://github.com/opencontainers/image-spec/blob/main/image-index.md
+## Supporting Clusters with Multi-Architecture Compute Nodes
+The [Fundamentals](#Fundamentals) above aim to guide authors on the key steps to building and distributing operators that can run on multiple architectures. These instructions are sufficient when your cluster's compute nodes share the same supported platform. However, operator authors should also understand the implications of running their operators in a cluster with multi-architecture compute nodes, since it is not always guaranteed that the architectures of the compute nodes will match the architectures supported by the operator.
+
+### Safe Scheduling Using Node Affinity
+Node affinity is a mechanism exposed in a Kubernetes pod template that allows a `PodSpec` author to instruct the scheduler to restrict a pod to run only on (or with preference for) nodes that meet specific criteria. In order to ensure that pods are always scheduled to nodes of compatible architecture, it is a best practice for authors to set node affinity requirements to ensure their operators and operands will only schedule to the platforms supported by the images in the pod. If you don't do this, a container scheduled to an incompatible node will immediately crash with an `exec format error`, which will ultimately lead to an `ImagePullBackoff` event as the pod is restarted only to crash again with the same error.
```

Comment:
`will only schedule to the platforms`
to
`will only schedule to the nodes with architectures available to the images in the pod`